spec/rubocop/cop/rspec/context_method_spec.rb

```diff
@@ -11,6 +11,13 @@
     RUBY
   end
 
+  it 'ignores context without `.` or `#` at the beginning' do
+    expect_no_offenses(<<-RUBY)
+      context '_foo_bar' do
```

Comment:
```suggestion
      context "when it's sunny" do
```
I think this would be closer to real-life usages